src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -16,14 +16,31 @@
 public class HelpWindow extends UiPart<Stage> {
 
     public static final String USERGUIDE_URL = "https://ay2021s1-cs2103t-t12-4.github.io/tp/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+
+    public static final String COMMON_COMMANDS = "Add a patient ---"
+            + "add n/NAME i/NRIC p/PHONE_NUMBER a/ADDRESS [t/TAG]…\n"
+            + "Edit a patient --- edit INDEX [n/NAME] [i/NRIC] [p/PHONE_NUMBER] [a/ADDRESS] [t/TAG]…\n"
```

Comment:
Perhaps the prefix [r/REMARK] can be added to the command syntax for both the add and edit commands.
I've just made this change in our user guide as well in my latest PR 👍🏻

Reply:
Ok I will add it in and update the helpMessage image